runtime/oti/util_api.h

```diff
@@ -2491,11 +2491,12 @@ gpProtectAndRun(protected_fn function, JNIEnv * env, void *args);
  * @param currentClass the class that contains the invokespecial
  * @param resolvedClass the class resolved from the MethodRef in the constantpool
  * @param method the J9Method returned by javaLookupMethod() for the MethodRef.
+ * @param extraOptions extra options in the case of re-resolution for super send
```

Comment:
`extraOptions` -> `options`.  I'm unclear on why this would be extra options rather than just an options variable.

Reply:
Thinking about this further - would `lookupOptions` be clearer as this is the options passed to lookupMethod()?

Reply:
Yes and no - we need to augment it with the no-visibility-check flag for the special resolve case, and the other cases don't have any local options.  I do not object to making it explicit in the callers as to what they want to do.

Reply:
I withdraw my objection to the name.  Given this function hardcodes some of the options, 'extra' is probably clear enough.